src/spec/junit_formatter.cr

```diff
@@ -41,14 +41,29 @@ module Spec
       JUnitFormatter.new(file)
     end
 
+    private def escape_xml_attr(value)
+      String.build do |io|
```

Comment:
Why not simply return `value.inspect_unquoted`?

Reply:
Because we just wanna quote control characters.

Reply:
Yeah, right. We could consider making `String#dump_or_inspect_unquoted` public for such use cases. But it's fine for now.
How about `HTML.escape`, though?

Reply:
> Yeah, right. We could consider making `String#dump_or_inspect_unquoted` public for such use cases. But it's fine for now.

👍

>  How about `HTML.escape`, though?

It is being escaped l8r on, these two things have nothin' in common.

Reply:
Why would `HTML.escape` not escape control characters??

Reply:
Because they have nuffin' to do with HTML...?

Reply:
Well `HTML.escape` really applies XML escaping, does it not?

Reply:
We need to escape control characters into their literals (`\n` => `\\n`), still nuffin' to do with XML/HTML escaping.

Reply:
Oh, my bad, I assumed you were escaping these as XML entities.